pkg/utils/utils.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"flag"
+	log "github.com/Sirupsen/logrus"
```

Comment:
`libcalico-go/lib/client/client.go` (libcalico-go v1.5.1) is using `"github.com/Sirupsen/logrus"`. 
```
vendor/github.com/projectcalico/libcalico-go/lib/client/client.go:24:2: cannot find package "github.com/Sirupsen/logrus" in any of:
	/root/go/src/github.com/projectcalico/k8s-policy/vendor/github.com/Sirupsen/logrus (vendor tree)
	/opt/go/src/github.com/Sirupsen/logrus (from $GOROOT)
	/root/go/src/github.com/Sirupsen/logrus (from $GOPATH)
```	
If I start importing "github.com/sirupsen/logrus" (lower case), it starts giving import collisions. 

```
main.go:8:2: case-insensitive import collision: "github.com/projectcalico/k8s-policy/vendor/github.com/Sirupsen/logrus" and "github.com/projectcalico/k8s-policy/vendor/github.com/sirupsen/logrus"
```
To avoid this, I also started aligning to "github.com/Sirupsen/logrus"